src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,123 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import core.basesyntax.service.ValidationException;
+import java.util.Random;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
```

Comment:
Let's stick to some rules:
one test method - one test case, if you would like to check several parameters use ParameterizedTest
Tests should be isolated, don't need to check null, "" and min length in one test method. 
Also, name of test method should describe what you are going to test - register_validUser_ok, 
register_nullPassword_notOk etc